### ISSUE TITLE
train ASR task with EAT model

### ISSUE THREAD

**Issue by qiuqiu-879** [X-LANCE/SLAM-LLM#238]
Hi,  I would like to ask if you have tried to train ASR with the EAT model, I directly used AAC code to train, and use a small llm Qwen2.5 0.5B and the effect is not good , wer is 50 or higher,  I use other encoder like Whisper / Wavlm, the result is good after about 2-3 epoch ,  dataset is Librispeech, I wonder if it is because the EAT has not been trained on librispeech or other reasons . Thanks !

**Comment by qiuqiu-879** [comment]
Thanks for your reply! I wanted to do some ASR tasks on private datasets while many of it without correct labels, and I'm wondering if I can achieve good performance only pre-train on speech data without doing classification finetune task?
